packages/messaging/android/src/main/java/io/invertase/firebase/messaging/ReactNativeFirebaseMessagingModule.java

```diff
@@ -120,8 +121,9 @@ public void setAutoInitEnabled(Boolean enabled, Promise promise) {
   }
 
   @ReactMethod
-  public void getToken(Promise promise) {
-    Tasks.call(getExecutor(), () -> Tasks.await(FirebaseMessaging.getInstance().getToken()))
+  public void getToken(String projectId, String senderId, Promise promise) {
+    FirebaseMessaging messagingInstance = FirebaseApp.getInstance(projectId).get(FirebaseMessaging.class);
```

Comment:
I think actually the getInstance expects an app name, not the projectId. So probably it is better to change that variable name (https://firebase.google.com/docs/reference/admin/java/reference/com/google/firebase/FirebaseApp#getInstance(java.lang.String))